### ISSUE TITLE
Broken on the latest comfy

### ISSUE THREAD

**Issue by ponchojohn1234** [ssitu/ComfyUI_restart_sampling#9]
breaks if the restart scheduler is ddim_uniform, simple or simple_test

```
Error occurred when executing KRestartSampler:

'KSamplerX0Inpaint' object has no attribute 'sigma_to_t'

  File "F:\AI\ComfyUI\execution.py", line 153, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "F:\AI\ComfyUI\execution.py", line 83, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "F:\AI\ComfyUI\execution.py", line 76, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\nodes.py", line 82, in sample
    return restart_sampling(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, segments, restart_scheduler, denoise=denoise)
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\restart_sampling.py", line 95, in restart_sampling
    samples = common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, denoise=denoise,
  File "F:\AI\ComfyUI\nodes.py", line 1269, in common_ksampler
    samples = comfy.sample.sample(model, noise, steps, cfg, sampler_name, scheduler, positive, negative, latent_image,
  File "F:\AI\ComfyUI\comfy\sample.py", line 100, in sample
    samples = sampler.sample(noise, positive_copy, negative_copy, cfg=cfg, latent_image=latent_image, start_step=start_step, last_step=last_step, force_full_denoise=force_full_denoise, denoise_mask=noise_mask, sigmas=sigmas, callback=callback, disable_pbar=disable_pbar, seed=seed)
  File "F:\AI\ComfyUI\comfy\samplers.py", line 711, in sample
    return sample(self.model, noise, positive, negative, cfg, self.device, sampler, sigmas, self.model_options, latent_image=latent_image, denoise_mask=denoise_mask, callback=callback, disable_pbar=disable_pbar, seed=seed)
  File "F:\AI\ComfyUI\comfy\samplers.py", line 617, in sample
    samples = sampler.sample(model_wrap, sigmas, extra_args, callback, noise, latent_image, denoise_mask, disable_pbar)
  File "F:\AI\ComfyUI\comfy\samplers.py", line 556, in sample
    samples = self.sampler_function(model_k, noise, sigmas, extra_args=extra_args, callback=k_callback, disable=disable_pbar, **self.extra_options)
  File "F:\AI\ComfyUI\venv\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\restart_sampling.py", line 137, in ksampler_restart_wrapper
    seg_sigmas = calc_sigmas(_restart_scheduler, n_restart, s_min,
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\restart_sampling.py", line 60, in calc_sigmas
    return SCHEDULER_MAPPING[scheduler](model, n, sigma_min, sigma_max, device)
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\restart_schedulers.py", line 52, in get_sigmas_ddim_uniform
    t_min, t_max = model.sigma_to_t(torch.tensor([s_min, s_max], device=device))
  File "F:\AI\ComfyUI\venv\lib\site-packages\torch\nn\modules\module.py", line 1614, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
```

**Comment by ponchojohn1234** [comment]
KRestartSamplerSimple also error if using sgm_uniform
```
Error occurred when executing KRestartSamplerSimple:

'sgm_uniform'

  File "F:\AI\ComfyUI\execution.py", line 153, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "F:\AI\ComfyUI\execution.py", line 83, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "F:\AI\ComfyUI\execution.py", line 76, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\nodes.py", line 54, in sample
    return restart_sampling(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, segments, scheduler, denoise=denoise)
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\restart_sampling.py", line 95, in restart_sampling
    samples = common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, denoise=denoise,
  File "F:\AI\ComfyUI\nodes.py", line 1269, in common_ksampler
    samples = comfy.sample.sample(model, noise, steps, cfg, sampler_name, scheduler, positive, negative, latent_image,
  File "F:\AI\ComfyUI\comfy\sample.py", line 100, in sample
    samples = sampler.sample(noise, positive_copy, negative_copy, cfg=cfg, latent_image=latent_image, start_step=start_step, last_step=last_step, force_full_denoise=force_full_denoise, denoise_mask=noise_mask, sigmas=sigmas, callback=callback, disable_pbar=disable_pbar, seed=seed)
  File "F:\AI\ComfyUI\comfy\samplers.py", line 711, in sample
    return sample(self.model, noise, positive, negative, cfg, self.device, sampler, sigmas, self.model_options, latent_image=latent_image, denoise_mask=denoise_mask, callback=callback, disable_pbar=disable_pbar, seed=seed)
  File "F:\AI\ComfyUI\comfy\samplers.py", line 617, in sample
    samples = sampler.sample(model_wrap, sigmas, extra_args, callback, noise, latent_image, denoise_mask, disable_pbar)
  File "F:\AI\ComfyUI\comfy\samplers.py", line 556, in sample
    samples = self.sampler_function(model_k, noise, sigmas, extra_args=extra_args, callback=k_callback, disable=disable_pbar, **self.extra_options)
  File "F:\AI\ComfyUI\venv\lib\site-packages\torch\utils\_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\restart_sampling.py", line 137, in ksampler_restart_wrapper
    seg_sigmas = calc_sigmas(_restart_scheduler, n_restart, s_min,
  File "F:\AI\ComfyUI\custom_nodes\ComfyUI_restart_sampling\restart_sampling.py", line 60, in calc_sigmas
    return SCHEDULER_MAPPING[scheduler](model, n, sigma_min, sigma_max, device)
```
